packages/@aws-cdk/aws-logs/README.md

```diff
@@ -32,6 +32,13 @@ retention period (including infinite retention).
 
 [retention example](test/example.retention.lit.ts)
 
+**Note** When using `LogRetention` to configure retention period for the log group,
```

Comment:
This was missed in the previous PR - can we add a separate section in the README dedicated to the `LogRetention` construct? 

I suspect it will look something similar to - https://github.com/aws/aws-cdk/tree/master/packages/@aws-cdk/aws-lambda#singleton-function

Reply:
Added a new entry